src/main/resources/categories.json

```diff
@@ -0,0 +1 @@
+["you"]
```

Comment:
remove this folder and file, it should be under article-analyzer/src/main/resoures

Reply:
done